client/.eslintrc

```diff
@@ -16,7 +16,7 @@
     "no-catch-shadow": 0,
     "no-class-assign": 0,
     "no-cond-assign": 2,
-    "no-console": 2,
+    "no-console": 1,

```

Comment:
Moving this to a warning. On error nothing will render in the UI. There's a way to suppress errors during runtime but I'll have to look into it.